cloudvolume/paths.py

```diff
@@ -117,7 +117,8 @@ def extract(cloudpath, windows=None, disable_toabs=False):
   if len(cloudpath) == 0:
     return ExtractedPath('','','','','','','')
 
-  windows_file_re = re.compile(r'((?:\w:\\)[\d\w_\.\-]+(?:\\)?)') # for C:\what\a\great\path
+  # windows_file_re = re.compile(r'((?:\w:\\)[\d\w_\.\-]+(?:\\)?)') # for C:\what\a\great\path
+  windows_file_re = re.compile(r'((?:\w:\\)[\d\w_\.\-]+)')
```

Comment:
`(?:\\)?)` allows the paths to end with a slash

Reply:
If `(?:\\)?)` is included, `bucket` gets set to `C:\\wow\\` when it should be `C:\\wow` right? So when basepath gets created like:
```python
basepath = split_char.join([bucket] + splitties[1:-1])
```
It has too many `\\` characters between bucket and the rest of the path:
`C:\\wow\\\\this\\is\\a\\cool`

With the updated regex, a cloudpath ending with a `\\` will still be interpreted correctly in windows. I added a check for this in the new test function.

Reply:
I see. Would you mind removing the commented out line? Thanks for your help!

Reply:
sure.